.gitignore

```diff
@@ -64,6 +64,7 @@ python/lib/pyspark.zip
 python/.eggs/
 python/deps
 python/docs/_site/
+python/docs/source/reference/api/
```

Comment:
This is generated by `autosummary` plugin in Sphinx when `autosummary_generate` in `conf.py` is enabled. Each page of API or class under `autosummary` directive, for example, [`DataFrame.alias`](https://hyukjin-spark.readthedocs.io/en/stable/reference/api/pyspark.sql.DataFrame.alias.html#pyspark.sql.DataFrame.alias) will be generated via that plugin as RST files.